src/prefect/cli/server.py

```diff
@@ -501,3 +509,286 @@ async def stamp(revision: str):
+
+
+def _is_process_running(pid: int) -> bool:
+    """Check if a process is running by attempting to send signal 0."""
+    try:
+        os.kill(pid, 0)
+        return True
+    except (ProcessLookupError, OSError):
+        return False
+
+
+def _read_pid_file(path: Path) -> int | None:
+    """Read and validate a PID from a file."""
+    try:
+        return int(path.read_text())
+    except (ValueError, OSError, FileNotFoundError):
+        return None
+
+
+def _write_pid_file(path: Path, pid: int) -> None:
+    """Write a PID to a file, creating parent directories if needed."""
+    path.parent.mkdir(parents=True, exist_ok=True)
+    path.write_text(str(pid))
+
+
+def _cleanup_pid_file(path: Path) -> None:
+    """Remove PID file and try to cleanup empty parent directory."""
+    path.unlink(missing_ok=True)
+    try:
+        path.parent.rmdir()
+    except OSError:
+        pass
+
+
+# this is a hidden command used by the `prefect server services start --background` command
+@services_app.command(hidden=True, name="manager")
+def run_manager_process():
+    """
+    This is an internal entrypoint used by `prefect server services start --background`.
+    Users do not call this directly.
+
+    We do everything in sync so that the child won't exit until the user kills it.
+    """
+    if not (enabled_services := _get_enabled_services()):
+        logger.error("No services are enabled! Exiting manager.")
+        sys.exit(1)
+
+    logger.info("Manager process started. Starting services...")
+    try:
+        asyncio.run(_run_services(enabled_services))
+    except KeyboardInterrupt:
+        pass
+    finally:
+        logger.info("Manager process has exited.")
+
+
+# public, user-facing `prefect server services` commands
+@services_app.command(aliases=["ls"])
+def list_services():
+    """List all available services and their status."""
+    service_settings = _get_service_settings()
+    table = Table(title="Available Services", expand=True)
+    table.add_column("Name", style="blue", no_wrap=True)
+    table.add_column("Enabled?", style="green", no_wrap=True)
+    table.add_column("Description", style="cyan", no_wrap=False)
+
+    for svc in _discover_service_classes():
+        name = svc.__name__
+        setting = service_settings.get(name, False)
+        is_enabled = setting.value() if setting else False  # type: ignore
+        assert isinstance(is_enabled, bool), "Setting value is not a boolean"
+
+        doc = inspect.getdoc(svc) or ""
+        description = doc.split("\n", 1)[0].strip()
+        table.add_row(name, str(is_enabled), description)
+
+    app.console.print(table)
+
+
+@services_app.command(aliases=["start"])
+def start_services(
+    background: bool = typer.Option(
+        False, "--background", "-b", help="Run the services in the background"
+    ),
+):
+    """Start all enabled Prefect services in one process."""
+    pid_file = Path(PREFECT_HOME.value()) / "services" / "manager.pid"
```

Comment:
two minor points:
- given this is an important file and a global, I suggest initializing it (but not creating it) at the top of the file and calling it `PID_FILE`
- to mirror the `server start` command, I think it might be more consistent to use `Path(PREFECT_HOME.value()) / "services.pid"` rather than creating a new subdirectory